app/lib/constants/colors.ts

```diff
@@ -110,6 +110,8 @@ export const colors = {
 		strokeExtraLight: '#EBECEF',
 		strokeLight: '#CBCED1',
 		surfaceTint: '#F7F8FA',
+		fontTitlesLabels: '#1F2329',
```

Comment:
Where are you using this color?

Reply:
Nice catch! This color was related to previous design!